presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -146,7 +146,8 @@ class TaskManager {
       const std::string& nodeIp,
       const std::string& nodeId,
       const std::string& queryId,
-      const protocol::TaskId& taskId);
+      const protocol::TaskId& taskId,
+      const bool includeNodeInSpillPath);
```

Comment:
```
bool includeNodeInSpillPath
```
bool is passed by value and no need for const annotation.

Reply:
Thanks. Fixed.